.docs/user-guide/automation.md

```diff
@@ -14,7 +14,32 @@ orchestration tools below.  Optionally, Docker is also listed in some examples.
 
 <br>
 ## Ansible
-ToDo
+With Ansible, installing the latest REX-Ray binaries can be accomplished by
+including the `codenrhoden.rexray` role from Ansible Galaxy.  The role accepts
+all the necessary variables to properly fill out your `config.yml` file.
+
+Install the role from Galaxy:
+
+```shell
+ansible-galaxy install codenrhoden.rexray
+```
+
+Example playbook for installing REX-Ray on GCE Docker hosts:
+
+```yaml
+- hosts: gce_docker_hosts
+  roles:
+  - { role: codenrhoden.rexray,
```

Comment:
Hi @codenrhoden,

Is this valid YAML? I've never seen YAML with a leading and ending curly bracket. Is this supposed to be a map? If so, isn't a map in YAML just another level of key/value pairs? You know what, never mind, I just found that's a way to [represent a mapping](http://symfony.com/doc/current/components/yaml/yaml_format.html).